### ISSUE TITLE
QueryStringSerializer and null/empty strings doesn't work as expected

### ISSUE THREAD

**Issue by JBTuu5** [ServiceStack/Issues#552]
I am trying to use QueryStringSerializer.SerializeToString to serialize a DTO:
```
public class StripeCreateSubscription
{
    public string customer { get; set; }

    public Dictionary<string, string> metadata { get; set; }

    public string plan { get; set; }
}
```

..with the following code:
`QueryStringSerializer.ComplexTypeStrategy = QueryStringStrategy.FormUrlEncoded;
QueryStringSerializer.SerializeToString(new StripeCreateSubscription { metadata = new Dictionary<string, string> { { "First", string.Empty } } });`

This yields the following string:
`metadata[First]=""`

Replace string.Empty with null and it turns into this:
`metadata[First]=null`

Desired output:
`metadata[First]=`

How do I get that desired output? Is there some special keyword I am missing besides string.Empty and null? One would think one of those would output the desired result. I think there may be a bug with handling string.Empty - who would want that to turn into quotes?

**Comment by JBTuu5** [comment]
Looks like a fix would go [here](https://github.com/ServiceStack/ServiceStack.Text/blob/master/src/ServiceStack.Text/QueryStringSerializer.cs#L254).

I think null is implemented right, but perhaps add another if statement to handle an empty string before writeFn interprets it wrong?

**Comment by mythz** [comment]
Fix added in [this commit](https://github.com/ServiceStack/ServiceStack.Text/commit/7686fb11dd075d348cfaeed3bcc9b044c8eeca37), this change is available from v4.5.13 that's now [available on MyGet](http://docs.servicestack.net/myget).
